src/cohere/manually_maintained/tokenizers.py

```diff
@@ -12,39 +13,64 @@ def tokenizer_cache_key(model: str) -> str:
     return f"{TOKENIZER_CACHE_KEY}:{model}"
 
 
-async def get_hf_tokenizer(co: typing.Union["AsyncClient", "Client"], model: str) -> Tokenizer:
+def get_hf_tokenizer(co: "Client", model: str) -> Tokenizer:
+    """Returns a HF tokenizer from a given tokenizer config URL."""
+    tokenizer = co._cache_get(tokenizer_cache_key(model))
+    if tokenizer is not None:
+        return tokenizer
+    tokenizer_url = co.models.get(model).tokenizer_url
+    if not tokenizer_url:
+        raise ValueError(f"No tokenizer URL found for model {model}")
+
+    size = int(typing.cast(int, requests.head(tokenizer_url).headers.get("Content-Length")))
+    size_mb = round(size / 1024 / 1024, 2)
+    print(f"Downloading tokenizer for model {model}. Size is {size_mb} MBs.")
```

Comment:
Did we decide on print versus logging? Logging is a better approach, but I can see that we don't do much of it within this SDK

Reply:
I don't think we decided on anything. I saw that the SDK didn't have any logging setup, so I just went with print statements (since it's just progress indication for the user, not error/structured logs)

@billytrend-cohere do you have opinions on logging?

Reply:
I would recommend choosing logging over print, as people who use this library will likely want some level of control over how and when log lines are printed.